File: Server/controllers/wallet.js
const Wallet = require("../models/Wallet");

exports.updateWallet = async (req, res, next) => {
  const { balance, expenses, userid } = req.body;

  console.log(balance, expenses, userid);
  try {
    await Wallet.updateWallet(balance, userid, expenses);
    res.json({ success: true, message: "Wallet Uploaded" });
  } catch (error) {
    res.json({ success: false, message: error.message });
  }
};

File: DesktopClientApp/DesktopClientApp/Models/Wallet.cs
using System;
using System.Collections.Generic;

namespace DesktopClientApp.Models
{
    public class Wallet
    {
        public Wallet()
        {
            this.balance = 0;
            this.expenses = new List<Expense>();
        }
        public double balance { get; set; }
        public string token { get; set; }
        public int userid { get; set; }
        public List<Expense> expenses { get; set; }
    }
}
